snmp/datadog_checks/snmp/data/profiles/apc_ups.yaml

```diff
@@ -8,6 +8,50 @@ device:
 
 sysobjectid: 1.3.6.1.4.1.318.1.*
 
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "apc"
+      serial_number:
+        symbol:
+          MIB: POWERNET-MIB
+          OID: 1.3.6.1.4.1.318.1.1.1.1.2.3.0
+          name: upsAdvIdentSerialNumber
+          # The serial number of the UPS internal microprocessor.  This number is set at
+          # the factory. NOTE: This number does NOT correspond to the serial number on the rear of the UPS.
+          # Examples: NA0835009156, AQ1805133700
+      version:
+        symbol:
+          OID: 1.3.6.1.4.1.318.1.1.1.1.2.1.0
+          name: upsAdvIdentFirmwareRevision
+      product_name:
+        symbol:
+          OID: 1.3.6.1.4.1.318.1.1.1.1.1.1.0
+          name: upsBasicIdentModel
+      model:
+        symbol:
+          OID: 1.3.6.1.2.1.1.1.0
+          name: sysDescr
+          match_pattern: 'MN:(\w+)'
+          match_value: '$1'
+          # Examples: MN:AP9631
```

Comment:
Could we have an comment with an example of a sysDescr we expect? Will be easier if we ever have to come back to that profile

Reply:
+1, added here https://github.com/DataDog/integrations-core/pull/10857/commits/e409eeaae1e4686a40cbf901a89fc2bcacdb2d35